libsnark/gadgetlib1/gadgets/hashes/anemoi/tests/anemoi_outputs.cpp

```diff
@@ -0,0 +1,111 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libff, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+#ifndef LIBSNARK_GADGETLIB1_GADGETS_HASHES_ANEMOI_TESTS_ANEMOI_OUTPUTS_CPP_
```

Comment:
Should not need these guards in cpp files.

Reply:
Removed